mixer/cmd/mixc/cmd/root.go

```diff
@@ -79,9 +79,9 @@ func addAttributeFlags(cmd *cobra.Command, rootArgs *rootArgs) {
 	cmd.PersistentFlags().IntVarP(&rootArgs.repeat, "repeat", "r", 1,
 		"Sends the specified number of requests in quick succession")
 	cmd.PersistentFlags().IntVarP(&rootArgs.concurrency, "concurrency", "c", 1,
-		"Number of concurrent goroutines sending requests to Mixer.")
+		"Controls the number of workers to send requests to Mixer conrrently.")
```

Comment:
Controls the numbers of workers used to send requests to Mixer concurrently.

Reply:
Done.